docs/metrics.md

```diff
@@ -267,6 +267,39 @@ Returns the labels of the persistent volume claims that are used for restoring v
 ### kubevirt_vnc_active_connections
 Amount of active VNC connections, broken down by namespace and vmi name. Type: Gauge.
 
+### reflector_items_per_list
```

Comment:
@machadovilaca We are not supposed to document non kubevirt metrics. This applies to all metrics that were added here. Let's discuss this.